packages/doc/babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  plugins: ['@babel/plugin-proposal-optional-chaining', 'import-glob'],
```

Comment:
shouldn't we use [gatsby-plugin-optional-chaining](https://www.gatsbyjs.org/packages/gatsby-plugin-optional-chaining/) here?

Reply:
Removed!